regulatory/.gitignore

```diff
@@ -0,0 +1,2 @@
+tmp
+release/*.pdf
```

Comment:
I was confused why you're not ignoring all of `release`/.  Is the intent to check in the release Markdown and JSON files, but not the PDFs?

If so, can we write that down somewhere in here?

Reply:
I'm guessing PDFs are larger files and are just generated to send directly to people. The same information is likely generated in the other formats like the markdown

Reply:
According to the documentation:
"The PDF versions are generated for submission to regulatory bodies or for upload to other document control systems."

So they essentially aren't supposed to be uploaded your GitHub. Rather they serve other purposes.

Reply:
That is correct. Since the PDF files are directly generated from markdown, there is no need to also commit them into version control.

You are correct the PDFs serve other purposes. The FDA, for example, requires submitting the RDM artifacts as PDFs. Auditors also will probably prefer reading PDFs over markdown.